config/secrets.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
This file should be supplied in the same manner as the founders json file, via environment variable that specifies the path.

Reply:
I can code that if you wish